src/switch_core_media.c

```diff
@@ -11894,6 +11987,7 @@ SWITCH_DECLARE(void) switch_core_media_gen_local_sdp(switch_core_session_t *sess
 		}
 
 		smh->mparams->rtcp_audio_interval_msec = SWITCH_RTCP_AUDIO_INTERVAL_MSEC;
+		smh->mparams->partner_rtcp_audio_interval_msec = SWITCH_RTCP_AUDIO_INTERVAL_MSEC;
```

Comment:
We shouldn't initialize this right and should be 0 by default to prevent this profile updating the other channel profile when no profile configuration is set.